### DISCUSSION TITLE
Correcting a wrong charge-cost

### DISCUSSION THREAD

**Discussion by chicagoandy** [teslamate-org/teslamate#2175] [Q&A]
I misplaced a decimal when I created a geo-fence, and all my previous charges were calculated with the wrong cost.

I've fixed the geo-fence, but the new costs have not been applied to the previous charges.   I did not get the modal asking if I wanted to do this.

How do I kick the function to recalculate the historical charges?

Would zeroing out the Cost field in Charging_processes be a good start?

**Comment by chicagoandy** [comment]
Update:  I felt brave.

Update Charging_processes set cost=cost/100 did what i needed.

**Reply by boba51** [reply]
Hi,
Sorry where did you enter this, I would like to do the same for 200+ home charges I put the wrong rate in for?

**Reply by chicagoandy** [reply]
I used a SQL tool (PGAdmin) to write an update statement against the charging_processes table in the TeslaFI Postgres database.

**Reply by boba51** [reply]
Thank you, sounds a little more complicated, I probably need to read up some more to understand it more.
